SpecFlowProject10/Configuration/Configuration.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.IO;
+using NLog;
+
+namespace Test.Configuration
+
+{
+
+    public class Configuration
+
+    {
+        private static readonly ILogger Logger = LogManager.GetCurrentClassLogger();
+        static string[] configFileLines;
+        public static string path(string appname)
+        {
+            //string configFilePath = "C:\\Users\\conspol\\source\\repos\\final\\SpecflowCalculator\\SpecFlowProject10\\Configuration\\AppConfig.txt";
+            //configFileLines = File.ReadAllLines(configFilePath);
+
+            //var app1Path = GetPath(appname);
+            /*for (int i = 0; i < configFileLines.Length; i++)
+            {
+                string lines = configFileLines[i];
+                Console.WriteLine($"configline is {lines}");
+            } */
+
+            //var app1WindowName = GetWindowName(appname);
+            //return app1Path;
+
+            try
+            {
+                string configFilePath = "C:\\Users\\conspol\\source\\repos\\final\\SpecflowCalculator\\SpecFlowProject10\\Configuration\\AppConfig.txt";
```

Comment:
You can use Directory.GetCurrentDirectory() here

Reply:
Done